tests/python_test.py

```diff
@@ -61,22 +61,22 @@ def test_write_lockspace(tmpdir, sanlock_daemon, filename, encoding, size, offse
     util.create_file(path, size)
 
     # Test read and write with default alignment and sector size values.
-    sanlock.write_lockspace("name", path, offset=offset, iotimeout=1)
+    sanlock.write_lockspace(b"name", path, offset=offset, iotimeout=1)
```

Comment:
lockspace and resource name should be parametric:
- "\xd7\x90" (py2, py3)
- u"ascii" (py2 compatibility)

We can do this for the same tests we use parametric paths.

Reply:
I posted this incomplete patch as example of how we should test this:
https://github.com/nirs/sanlock/commit/ff84992729c571db99b933d76226d1befec50013

@aabawer, what do you think?